ownCloud/Resources/en.lproj/Localizable.strings

```diff
@@ -80,6 +80,11 @@
 "Are you sure you want to delete this item from the server?" = "Are you sure you want to delete this item from the server?";
 "Deleting '%@'" = "Deleting '%@'";
 "Renaming to %@" = "Renaming to %@";
+"Upload from your photo library" = "Upload from your photo library";
+"ownCloud does not have permissons" = "ownCloud does not have permissons";
```

Comment:
Little typo here. "permissons" vs. "permissions".

With branding in mind… maybe turn this into "Missing permissions"?

(also at line 700 of ClientQueryViewController.swift)